inductiva/utils/format_utils.py

```diff
@@ -57,6 +57,30 @@ def apply_formatters(table_data: dict, formatters: dict):
     return table_data
 
 
+def get_tabular_data(
```

Comment:
missing doc string.
You can move part of the docstring from `get_tabular_str` here (making sure to update what needs to be updated) and reference this function in `get_tabular_str` so that there is no need to have doc repetition.